tests/data/docker/docker-compose.yml

```diff
@@ -2,17 +2,18 @@ version: '3'
 
 services:
   db:
-    image: postgres:9.6
+    # Change the image to try other versions: postgres:10, postgres:11, postgres:12, postgres:13
```

Comment:
We aren't using it, are we? https://github.com/yiisoft/db-pgsql/pull/29/files#diff-898e737ee4e56ba042bda31764bed49eR26

Reply:
This docker file only uses for local tests, we can delete it I have no problems.

Reply:
Let's remove it. It's better to run tests locally via something like https://github.com/nektos/act

Reply:
Problem in windows https://github.com/nektos/act/issues/145

Reply:
`necktos/act` It doesn't fix running the tests with github actions services locally, so I'll add a note that you must have an active connection to run the tests